src/calendar/day/basic/index.tsx

```diff
@@ -148,6 +148,32 @@ export default class BasicDay extends Component<BasicDayProps> {
     const {customStyles, selectedTextColor} = this.marking;
     const style = [this.style.text];
 
+    switch (this.props.date?.dayNameIndex) {
```

Comment:
I don't like this switch. Please try string formatting. Example: `style.push(this.style[`dayTextAtIndex${index}`]);`